AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyResultTransformers.swift

```diff
@@ -67,6 +67,96 @@ class IdentifyResultTransformers {
 
     }
 
+    static func processChild(ids: [String],
+                             blockMap: [String: AWSTextractBlock]) -> String {
+        var keyText = ""
+        for keyId in ids {
+            let keyBlock = blockMap[keyId]
+            guard let keyBlockType = keyBlock?.blockType else {
+                continue
+            }
+            switch keyBlockType {
+            case .word:
+                if let text = keyBlock?.text {
+                    keyText += text + " "
+                }
+            default: break
+            }
+        }
+        return keyText
+    }
+
+    static func processValue(ids: [String],
+                             blockMap: [String: AWSTextractBlock]) -> (String, Bool) {
+        var valueText = ""
+        var valueSelected = false
+
+        // VALUE block has a CHILD list of IDs for the WORD block
+        for valueId in ids {
+            let valueBlock = blockMap[valueId]
+            guard let valueRelatioins = valueBlock?.relationships else {
+                continue
+            }
+            for valueRelation in valueRelatioins {
+                guard let ids = valueRelation.ids else {
```

Comment:
for valueBlockRelation in valueBlockRelationships {
                guard let ids = valueBlockRelation.ids else {